docs/admin/disruptions.md

```diff
@@ -0,0 +1,87 @@
+---
+assignees:
+- mml
+
+---
+This guide is for anyone wishing to specify safety constraints on pods or anyone
+wishing to write software (typically automation software) that respects those
+constraints.
+
+* TOC
+{:toc}
+
+## Rationale
+
+Various cluster management operations may "voluntarily" evict pods. By
```

Comment:
I went more directly definitional for the first sentence.

For the second one, the given example is just that, an example.  It's not the definition of non-voluntary and only illustrates the principle.